### PR TITLE
Opensearch

### PR DESCRIPTION
**Pull request by JasonTheMain**
## Changes proposed in this pull request:

- changed elasticsearchservice to opensearchservice
- changed elasticsearchapi calls to opensearchapi calls
- changed elasticsearch instance types to opensearch api types

## Security considerations

None as this only allows the latest elastisearch and doesnt sway from previous code

### CODE REVIEW

File: catalog-template.yml [hunk 1]
```diff
@@ -1343,9 +1343,9 @@ elasticsearch:
         unit: "MONTHLY"
       displayName: "non-prod single node elasticsearch 6.8"
     free: false
-    elasticsearchVersion: 6.8
+    elasticsearchVersion: Elasticsearch_6.8
```
Comment (markdboyd) [inline catalog-template.yml:1346]: So with these changes, going forward services with these plans will be created using AWS OpenSearch, which supports versions of Elasticsearch as well?

But existing service instances will continue to run on the old Elasticsearch service and instance types?

File: services/rds/broker.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ type Options struct {
 	PubliclyAccessible    bool   `json:"publicly_accessible"`
 	Version               string `json:"version"`
 	BackupRetentionPeriod int64  `json:"backup_retention_period"`
-        RedisVersion          string 'json:"redis_version"
```
Comment (markdboyd) [inline services/rds/broker.go:27]: I think the changes in this file are unrelated and shouldn't be in this PR?

File: catalog-template.yml (hunk 1 above)
Comment (JasonTheMain) [reply catalog-template.yml:1346]: the instance types is now a generic version of search instead of just for elasticsearch but yes, opensearch supports the latest elasticsearch versions and opensearch.

File: services/rds/broker.go (hunk 2 above)
Comment (JasonTheMain) [reply services/rds/broker.go:27]: This kind of scrated my head and seems like technical debt from the merge before. this shouldn't be there and caused an error when ran. I'm not sure how this passed through but this corrects that mistake.

Comment (markdboyd) [reply services/rds/broker.go:27]: I think this is because you need to update this branch with the latest from `main`

**Review by dandersonsw — APPROVED** [review]
LGTM
